source/simulator/assembly.cc

```diff
@@ -1286,6 +1286,20 @@ namespace aspect
           return residuals;
         }
 
+        double
+        dg_approximation_to_face_measure(const typename DoFHandler<2>::face_iterator face) const
+        {
+          return (face->vertex(0)-face->vertex(1)).norm();
+        }
+
+        double
+        dg_approximation_to_face_measure(const typename DoFHandler<3>::face_iterator face) const
+        {
+          const Tensor<1,3> v03 = face->vertex(3) - face->vertex(0);
+          const Tensor<1,3> v12 = face->vertex(2) - face->vertex(1);
+          Tensor<1,3> twice_area = cross_product_3d(v03, v12);
```

Comment:
make it `const`. If that's how it is in deal.II, want to submit your first deal.II patch as well? ;-)